packages/app/obojobo-repository/shared/components/button.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../client/css/defaults';
+
+.repository--button {
+	border: 0;
+	background: $color-button-bg;
+	color: white;
+	border-radius: 4px;
+	padding: 8px 18px;
+	font-weight: bold;
+	box-sizing: border-box;
+	cursor: pointer;
+	text-decoration: none;
+	font-size: 11px;
+
+	&:hover {
+		background: $color-action;
+	}
+
+	&.close-button {
+		$size: 25px;
```

Comment:
There's a ton of these.  I did a few, but there's more.  I could see potentially ignoring page-homepage.scss just because it's a cheap import from obojobo classic.